packages/core/src/Drawer/Drawer.tsx

```diff
@@ -14,6 +14,7 @@ export type PlacementOptions = 'top' | 'bottom' | 'right' | 'left'
 export type DrawerProps = SpaceProps &
   LayoutProps & {
     children?: string | React.ReactNode
+    customHeader?: string | React.ReactNode
```

Comment:
`React.ReactNode` allows string, no need to add it explicitly in the union. Also, feel free to clean up the types for `children` and `heading` also

Reply:
Good point!